BunnyHopMaster/Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -140,7 +140,7 @@ private void CheckGrounded()
 
     private void CheckJump()
     {
-        if (grounded && Input.GetKey(HotKeyManager.instance.GetKeyFor(PlayerConstants.Jump)))
+        if (grounded && Input.GetKey(HotKeyManager.Instance.GetKeyFor(PlayerConstants.Jump)))
```

Comment:
Verbose: `Input.GetKey(HotKeyManager.Instance.GetKeyFor(PlayerConstants.Jump))`

Thoughts: `InputManager.GetKey(PlayerConstants.Jump)`

Reply:
will make a ticket to wrap this to make it cleaner